File: Java_etc/jdbc_correction/src/main/java/model/Adherent.java
package model;

public class Adherent {
	private Integer numeroAdherent;
	private String prenom;
	private String nom;

	public Adherent() {

	}

	/**
	 * @param numeroAdherent
	 * @param prenom
	 * @param nom
	 */
	public Adherent(Integer numeroAdherent, String prenom, String nom) {
		this.numeroAdherent = numeroAdherent;
		this.prenom = prenom;
		this.nom = nom;
	}

	/**
	 * @param prenom
	 * @param nom
	 */
	public Adherent(String prenom, String nom) {
		this.prenom = prenom;
		this.nom = nom;
	}

	public Integer getNumeroAdherent() {
		return numeroAdherent;
	}

	public void setNumeroAdherent(Integer numeroAdherent) {
		this.numeroAdherent = numeroAdherent;
	}

	public String getPrenom() {
		return prenom;
	}

	public void setPrenom(String prenom) {
		this.prenom = prenom;
	}

	public String getNom() {
		return nom;
	}

	public void setNom(String nom) {
		this.nom = nom;
	}

}

File: angularCorrectionThibault/src/app/model/adherent.ts
export class Adherent {

  constructor(private _id?: number, private _prenom?: string, private _nom?: string) {

  }


  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get prenom(): string {
    return this._prenom;
  }

  set prenom(value: string) {
    this._prenom = value;
  }

  get nom(): string {
    return this._nom;
  }

  set nom(value: string) {
    this._nom = value;
  }
}
